src/Console/Command/ProcessCommand.php

```diff
@@ -137,6 +139,7 @@ private function ensureSomeRectorsAreRegistered(): void
     private function processFiles(array $fileInfos): void
     {
         $this->symfonyStyle->title('Processing files');
+        $this->symfonyStyle->progressStart(count($fileInfos));
```

Comment:
So easy to add this functionality :tada:

Reply:
It was pain before. Gotta know the right people ;)